plottable.css

```diff
@@ -120,19 +120,32 @@ svg.plottable {
   stroke-width: 1px;
 }
 
-.plottable .drag-box {
-  fill: aliceblue;
-  opacity: .7;
-  stroke: #C0C8FF;
-}
-
-/* SelectionBox */
 .plottable .selection-box-layer .selection-area {
   fill: black;
   fill-opacity: 0.03;
   stroke: #CCC;
 }
-/* /SelectionBox */
+/* DragBoxLayer */
+.plottable .drag-box-layer.x-resizable .drag-edge-lr {
+  cursor: ew-resize;
+}
+.plottable .drag-box-layer.y-resizable .drag-edge-tb {
+  cursor: ns-resize;
+}
+
+.plottable .drag-box-layer.x-resizable.y-resizable .drag-corner-tl {
+  cursor: nwse-resize;
+}
+.plottable .drag-box-layer.x-resizable.y-resizable .drag-corner-tr {
+  cursor: nesw-resize;
+}
+.plottable .drag-box-layer.x-resizable.y-resizable .drag-corner-bl {
+  cursor: nesw-resize;
+}
+.plottable .drag-box-layer.x-resizable.y-resizable .drag-corner-br {
+  cursor: nwse-resize;
+}
+/* /DragBoxLayer */
```

Comment:
I remember you hearing concerns about using these invisible targets.  Is this the way to go about it or should we just set the cursor based on pixel position ourselves?